source/ui/acquaman/AMScanConfigurationViewHolder3.cpp

```diff
@@ -133,15 +138,13 @@ AMAction3* AMScanConfigurationViewHolder3::createMultipleScans()
 	if (view_){
 
 		const AMScanConfiguration *config = view_->configuration();
+
 		if (config) {
-			int iteration = iterationsBox_->value();
-			if (iteration > 1) {
-				AMLoopAction3 *loopAction = new AMLoopAction3(new AMLoopActionInfo3(iterationsBox_->value(), config->name(), config->description()));
-				loopAction->addSubAction(createScan());
-				return loopAction;
-			} else {
-				return createScan();
-			}
+
+			AMLoopAction3 *loopAction = new AMLoopAction3(new AMLoopActionInfo3(iterationsBox_->value(), config->name(), config->description()));
+			loopAction->addSubAction(createScan());
+
+			return loopAction;
```

Comment:
Just checking that I understand. Before, the view holder created either scan actions or loop actions, depending on the desired number of scans. With these changes, the scan configuration view holder now always produces loop actions for scans--a single scan is now a loop action of one scan?